cmd/buildah/commit.go

```diff
@@ -77,6 +77,10 @@ var (
 			Name:  "tls-verify",
 			Usage: "Require HTTPS and verify certificates when accessing the registry",
 		},
+		cli.BoolFlag{
```

Comment:
nit, please alphabetize this, s/b before quite, near line 55

Reply:
done